doc/import.rst

```diff
@@ -0,0 +1,85 @@
+Importing SpikeInterface
+========================
+
+SpikeInterface allows for the generation of powerful and reproducible spike sorting pipelines.
+Flexibility is built into the package starting from import to maximize the productivity of
+the developer and the scientist. Thus there are three ways that SpikeInterface and its components
+can be imported:
+
+
+Importing by Module
+-------------------
+
+Since each spike sorting pipeline involves a series of often repeated steps, many of the developers
+working on SpikeInterface recommend importing in a module by module fashion. This will allow you to
+keep track of your processing steps (preprocessing, postprocessing, quality metrics, etc.). This can
+be accomplished by:
+
+.. code-block:: python
+
+    import spikeinterface as si
```

Comment:
Based on the speed metrics @h-mayorquin supplied why don't we suggest

`import spikeinterface.core as si`

It has a slight speed advantage directly importing rather than doing without core? And it follows the syntax for all the other submodules?

Reply:
I think this is perfect!

Reply:
Done!